src/screens/AccountList.js

```diff
@@ -30,10 +29,11 @@ import Button from '../components/Button';
 import AccountsStore from '../stores/AccountsStore';
 import { accountId } from '../util/account';
 import PopupMenu from '../components/PopupMenu';
+import Identicon from '@polkadot/reactnative-identicon';
```

Comment:
import ordering

Reply:
Sure, I'll remove that when removing the identicons from the main screnn